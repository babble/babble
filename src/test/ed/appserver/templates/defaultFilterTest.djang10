{{ unknownVar|default:"should see me" }}
{{ unknownVar|default:a }}
{{ urlParam|urlencode }}
{% filter urlencode %}{{ urlParam }}{% endfilter %}

{{ date|date:"a,A,b,d,D,f,F,g,G,h,H,i,j,l,L,m,M,n,N,O,P,r,s,S,t,w" }}
a   ,A ,b  ,d ,D  ,f   ,F       ,g,G ,h ,H ,i ,j ,l     ,L    ,m ,M  ,n ,N   ,O    ,P        ,r                              ,s ,S ,t ,w

{{ "moo"|upper }}
{{ "MOO"|lower }}

{% if array|length_is:"5" %}should see me{% endif %}

{{ array|length }} 