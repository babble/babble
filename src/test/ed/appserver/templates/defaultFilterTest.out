should see me
A
%3F%5C%2F%7E%21.%2C
%3F%5C%2F%7E%21.%2C

p.m.,PM,dec,20,Sun,3:11,December,3,15,03,15,11,20,Sunday,False,12,Dec,12,Dec.,-0500,3:11 p.m.,Sun, 20 Dec 1981 15:11:37 -0500,37,th,31,0
a   ,A ,b  ,d ,D  ,f   ,F       ,g,G ,h ,H ,i ,j ,l     ,L    ,m ,M  ,n ,N   ,O    ,P        ,r                              ,s ,S ,t ,w